### ISSUE TITLE
MultiSelect - accessibility issue - no focus indicator visible on clear button

### ISSUE THREAD

**Issue by amortiz** [carbon-design-system/carbon#6855]
1. go to https://react.carbondesignsystem.com/?path=/story/multiselect--default
2. select a few elements
3. When we focus on the button to "Clear Selection" by keyboard, there is no focus indicator visible.

Accessibility checkpoint:  2.4.7 Focus Visible
（https://www.ibm.com/able/guidelines/ci162/focus_visible.html）

see screenshot:
![image](https://user-images.githubusercontent.com/10033254/93471758-64ae3c80-f8b9-11ea-8786-7a4852562344.png)

**Comment by amortiz** [comment]
@rjhenriquez any ETA on the fix? trying to close out our accessibility defects for the quarter and this one is getting attention. Thanks!
